common/src/main/java/org/apache/gravitino/dto/requests/ModelVersionLinkRequest.java

```diff
@@ -63,5 +92,15 @@ public void validate() throws IllegalArgumentException {
             !NumberUtils.isCreatable(alias), "alias must not be a number or a number string");
       }
     }
+
+    if (uris == null || uris.isEmpty()) {
+      Preconditions.checkArgument(StringUtils.isNotBlank(uri), "URI cannot be empty");
```

Comment:
Can you please add more description about this check, for example, "uri field cannot be empty if uris field is null or empty".